.env.example

```diff
@@ -5,4 +5,6 @@ GOOGLE_KEY='909701487377-k6kp24c9a0414aba9bmmp5jev64b4kr9.apps.googleusercontent
 GOOGLE_SECRET='0Aaznl7tOtzaWuyQxHh9zr2P'
 GITHUB_KEY='cf5569da9e7aab3d4b6'
 GITHUB_SECRET='906be7adc9e30c4252e9e8fa9c6ae222c71998a6'
-
+MYSQL_USER='fake'
```

Comment:
O que tu acha de renomear para `DB_USER`? Costuma ser mais genérico (no caso de trocar de banco de dados) e se não to enganado é o padrão no rails - pelo menos é o padrão adotado nas empresas que trabalho.

Reply:
Estava mesmo pensando nisso, tens razão.